include/menoh/menoh.h

```diff
@@ -34,6 +34,8 @@ extern "C" {
 /*! @ingroup vpt
  */
 enum menoh_dtype_constant {
+    menoh_dtype_int32,
+    menoh_dtype_int64,
```

Comment:
This changes the value of `menoh_dtype_float` and breaks binary compatibility.
It is better to add them after `menoh_dtype_float` or to specify values explicitly to all of `menoh_dtype_constant` values.

Reply:
I see. I'll change the order.

Reply:
done.

Reply:
thanks! :+1: